terragrunt/aws/cloudfront/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "origin_access_identity" {
```

Comment:
Will we need this value in other modules?  It doesn't hurt anything to have it as an output, but I'm just curious.